server/backends/disk_cache/disk_cache.go

```diff
@@ -409,12 +409,19 @@ func newPartition(id string, rootDir string, maxSizeBytes int64, useV2Layout boo
 	return p, nil
 }
 
+// fileRecord is the data struct we store in the LRU cache
 type fileRecord struct {
 	key       *fileKey
-	lastUse   int64
 	sizeBytes int64
 }
 
+// fileRecordWrapper is a wrapper for fileRecord that contains additional metadata that does not need to be
+// written in the value field to the LRU cache
+type fileRecordWrapper struct {
```

Comment:
The `wrapper` suffix does not convey any information. What about calling this something like `timestampedFileRecord`? Similarly for the functions below.

Reply:
Good call - done